jsons/1.12-rel.json

```diff
@@ -35,9 +35,28 @@
       "serverreq":true
     },
     {
-      "name": "jline:jline:2.13",
+      "name": "net.minecraftforge:log4j-server-fix:2.8.1",
       "url" : "http://files.minecraftforge.net/maven/",
-      "checksums" : [ "2d9530d0a25daffaffda7c35037b046b627bb171" ],
+      "checksums" : [ "TODO" ],
+      "clientreq":false,
+      "serverreq":true
+    },
+    {
+      "name": "org.jline:jline:3.3.1",
```

Comment:
Needs to be uploaded to Forge maven: http://search.maven.org/remotecontent?filepath=org/jline/jline/3.3.1/jline-3.3.1.jar